docs/user/cli-installation.rst

```diff
@@ -17,19 +17,53 @@ It is easiest to install the CLI using `homebrew <https://brew.sh/>`_:
   brew install renku
 
 
-Pip Script Installer (``pipsi``)
---------------------------------
+Isolated environments using ``pipx``
+------------------------------------
 
-You can use `pipsi <https://github.com/mitsuhiko/pipsi>`_ to isolate
-dependencies and to guarantee that there are no version conflicts. Make sure
-you have the ``pipsi`` command correctly installed and ``~/.local/bin`` is in
-your ``PATH``.
+Install and execute Renku in an isolated environment using ``pipx``.
+It will guarantee that there are no version conflicts with dependencies
+you are using for your work and research.
 
-.. code-block:: console
+.. note::
+
+  This is the method of installation in the renku docker images,
+  i.e. the default environment you use when you launch a JupyterLab session
+  via the renku browser interface.
+
+`Install pipx <https://github.com/pipxproject/pipx#install-pipx>`_
+and make sure that the ``$PATH`` is correctly configured.
+
+::
+
+    $ python3 -m pip install --user pipx
+    $ pipx ensurepath
```

Comment:
This part is not gonna work on every environment. After installation with command as follows, pipx is not on the path for example on alpine and it will result in following error: ```/bin/sh: pipx: not found```. 

To make things a bit easier to follow for non-unix experts let's add the whole output of pipx ensure path to the docs.

```
/root/.local/bin/pipx ensurepath

Note:
To finish installation, '/root/.local/bin' must be added to your PATH
environemnt variable.

To do this, add the following line to your shell
config file (such as ~/.bashrc if using bash):

    export PATH=/root/.local/bin:$PATH
```

Reply:
? most people aren't `root`